dev-haskell/hackage-security/hackage-security-0.6.0.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.5.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Hackage security library"
+HOMEPAGE="https://github.com/haskell/hackage-security"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
```

Comment:
I think we need to comment out the KEYWORDS here again, as the current version `cabal-install-3.0` requires `< 0.6`

Reply:
But why, won't portage do the right thing with the deps specified?

Reply:
It should resolve the conflicts automatically, albeit slowing things down a bit. We can leave this as is and if Portage gets upset we can deal with it then.

I've had to mask pandoc-related packages before due to Portage bailing out before resolving the conflicts.